tools/core_packages.txt

```diff
@@ -78,6 +78,7 @@ python3
 readline
 rtmpdump
 ruby
+samurai
```

Comment:
Maybe this shouldn't be in core.  We want to make it leaner, not fatter.

Reply:
Eventually we'll remove ninja as we transition from samurai to ninja, since they're both super small, would you mind allowing them for the time being?

Reply:
I think the goal should be to exclude all build dependencies in core.

Reply:
okay, let's work on that then!